EdgeReachTech/adgereachtech-web-bn
# Blog

What does this PR do?

This PR introduces the following routes for the blog management system:

- POST /blog/createBlog - Route to create a new blog post.
- PATCH /blog/updateBlog/:blogId - Route to update an existing blog post.
- DELETE /blog/deleteBlog/:blogId - Route to delete a blog post.

Description of Task to be completed?

- Implement the route to create a new blog post using blogRouter.post("/createBlog", isLoggedIn, validateBlog, blogController.createBlog).
- Implement the route to update an existing blog post using blogRouter.patch("/updateBlog/:blogId", isLoggedIn, blogController.updateBlog).
- Implement the route to delete a blog post using blogRouter.delete("/deleteBlog/:blogId", isLoggedIn, blogController.deleteBlog).

How should this be manually tested?

Create a Blog Post:

Send a POST request to /blog/createBlog with the necessary blog data.
Ensure the user is logged in and the blog data is valid.

Update a Blog Post:

Send a PATCH request to /blog/updateBlog/:blogId with the updated blog data.
Ensure the user is logged in and the blog ID is valid.

Delete a Blog Post:

Send a DELETE request to /blog/deleteBlog/:blogId.
Ensure the user is logged in and the blog ID is valid.
